docs/changelog.md

```diff
@@ -4,7 +4,7 @@
 
 # Changelog
 
-## Unreleased
+## 0.68.0 (May 2025)
 
 - Add `VF2PostLayout` to default compilation.
```

Comment:
Should we update the pytket version before doing the release?

Reply:
May as well do that yes...